deployer/src/deployer/analyze_pr.py

```diff
@@ -88,7 +88,7 @@ def post_about_deployment(build_directory: Path, **config):
     links = []
     for doc in get_built_docs(build_directory):
         url = mdn_url_to_dev_url(config["prefix"], doc["mdn_url"])
-        links.append(f"- <{url}>")
+        links.append(f"- {url}")
```

Comment:
Let's assemble the link here instead of in `mdn_url_to_dev_url()`:

```suggestion
        mdn_url = doc["mdn_url"]
        links.append(f"- [{mdn_url}]({url})")
```